astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,368 @@
+# 1. standard library imports
+
+from collections import OrderedDict
```

Comment:
If you are not reordering any dictionaries frequently then you should not be using `OrderedDict`.
```suggestion
```